embeddings/app/embeddings.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Where do we need this import?

Reply:
This is used to enable postponed evaluation of types, not exactly used as a package import but more of a behavior import.